### PR TITLE
Add new pipeline for "Segfault on filtered vector search"

### PR DESCRIPTION
**Pull request by etiennedi**

### CODE REVIEW

File: apps/segfault-on-filtered-vector-search/run.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+            if (
+                'result' in result
+                and 'errors' in result['result']
+                and 'error' in result['result']['errors']
+            ):
+                for message in result['result']['errors']['error']:
+                    logger.error(message['message'])
+
+def load_records(client: weaviate.Client, max_records=100000):
+    global random_words
```
Comment (dirkkul) [inline apps/segfault-on-filtered-vector-search/run.py:60]: pretty sure you do not need to do that as you've defined it at the top level

File: apps/segfault-on-filtered-vector-search/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+weaviate-client==3.6.0
```
Comment (dirkkul) [inline apps/segfault-on-filtered-vector-search/requirements.txt:1]: use the current one? You could also do `weaviate-client>=3.8.0 ` so it always picks the latest

**Review by dirkkul — APPROVED** [review]

File: apps/segfault-on-filtered-vector-search/run.py (hunk 1 above)
Comment (etiennedi) [reply apps/segfault-on-filtered-vector-search/run.py:60]: You're probably right, I'll try removing it and see if all is still green.

File: apps/segfault-on-filtered-vector-search/requirements.txt (hunk 2 above)
Comment (etiennedi) [reply apps/segfault-on-filtered-vector-search/requirements.txt:1]: Good point, I'll update.

**Pull request merged**